File: EJERCICIO_1/EJERCICIOSPHP/nightTime.php
<?php
class NightTime extends Bonus {
    public function __construct(SalaryBass $salaryBass, $percentage, $days) {
        parent::__construct($salaryBass, 1.75, $days); // Fixed percentage of 1.75 for night hours
    }

    public   function calculateBonus() {
        return $this->valueDay * $this->percentage * $this->days; // Calculate the bonus
    }
}
?>

File: EJERCICIO_1/EJERCICIOJAVA/nightTime.java
package EJERCICIOJAVA;

public class nightTime extends bonus {
    public nightTime(salaryBass salary, double porcentage, int days) {
        super(salary, 1.75, days); 
    }

    public double calculateBonus() {
        return valueDay * percentage * days; 
    }
 }
